runtime/common/src/main/java/edu/snu/nemo/runtime/common/plan/StageEdge.java

```diff
@@ -57,14 +56,14 @@
    * @param coder the coder for enconding and deconding.
    * @param isSideInput whether or not the edge is a sideInput edge.
    */
-  public PhysicalStageEdge(final String runtimeEdgeId,
-                           final ExecutionPropertyMap edgeProperties,
-                           final IRVertex srcVertex,
```

Comment:
Could you explain more on `srcVertex` and `dstVertex` of `StageEdge` in the Constructor comment?
For example like the following: 

> srcVertex: IRVertex in the source stage that this StageEdge connects from.
dstVertex: IRVertex in the destination stage that this StageEdge connects to.

Reply:
Sure, I've added some more comments as per your suggestion.